docs/source/tutorial_basic.rst

```diff
@@ -17,6 +17,8 @@ Its goal is to provide researchers:
 - many popular datasets available all in one place, with the ability to multi-task over them
 - seamless integration of Amazon Mechanical Turk for data collection and human evaluation
 
+    - *new:* integration with Facebook Messenger for an alternative UI for connecting agents
```

Comment:
add link?   I wouldn't say 'alternative', i  wouldn't say 'new' either because at some point in won't be new, we already have that in news i would just have a section on parlai messenger, summarizing the tutorial

Reply:
I linked to the tutorial for now, I think it's a separate enough topic from the basics to leave for the tutorial